blog/2025-09-30-conda-roadmap-q4-2025.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Conda CLI Roadmap Updates: Q4, 2025"
+slug: "conda-roadmap-q4-2025"
+authors: [dashagurova, danyeaw]
+tags: [conda, roadmap]
+description: "The Q4 2025 conda CLI roadmap highlights improvements coming next year, including lockfiles support, sharded repodata support, and better integration with PyPI."
+---
+
+*The Q4 2025 conda CLI roadmap highlights improvements coming soon, including lockfiles for reproducibility, sharded repodata support for faster installs, and better integration with PyPI.*
+
+<!-- truncate -->
+
+---
+
+With any large open source project, dozens of pull requests and issues are always in flight, which can make it hard to see the bigger picture. Starting this quarter, we will publish regular roadmap updates to give you a clear view of what the [conda maintainers](https://github.com/orgs/conda/teams/conda-maintainers) team from Anaconda and Quansight, together with the broader community, is building and what you can expect.
+
+The [Q4 2025 roadmap](https://github.com/orgs/conda/projects/22) highlights the features we are focused on delivering this quarter and early next year. These priorities came directly from community feedback across GitHub, forums, and events, and the roadmap will continue to evolve as new needs emerge. Don’t see something specific on the roadmap? Let us know by [opening an issue](https://github.com/conda/conda/issues), or help us out by [submitting a contribution](https://github.com/conda/conda/blob/main/CONTRIBUTING.md) via a pull request!
+
+
+---
+
+## Q4 2025 Roadmap Priorities
+
+### More Consistency: Input File Handling Refactor
+
+**What we heard:** Conda supports different input files like `environment.yml` and `requirements.txt`, but behavior can feel inconsistent. Switching between formats or mixing them can cause confusion and errors.
```

Comment:
```suggestion
**What we heard:** Conda supports different ways to define dependencies via file including `environment.yml` and `requirements.txt`, but behavior can feel inconsistent across these different file types. Switching between formats or mixing them can cause confusion and errors.
```

For this line, can we link to specific cases of confusion and errors? As a maintainer of this project, I'm not even sure what you're talking about. Let's try to clarify this.